bindings/cs/Scripts/build.cake

```diff
@@ -1,7 +1,7 @@
-#addin nuget:?package=Cake.FileHelpers&version=3.3.0
-#addin nuget:?package=Cake.Git&version=0.22.0
-#addin nuget:?package=Cake.Incubator&version=5.1.0
-#tool dotnet:?package=GitVersion.Tool&version=5.3.7
+#addin nuget:?package=Cake.FileHelpers&version=5.0.0
+#addin nuget:?package=Cake.Git&version=2.0.0
+#addin nuget:?package=Cake.Incubator&version=7.0.0
+#tool dotnet:?package=GitVersion.Tool&version=5.10.3
```

Comment:
Upgraded these nuget packages to the same versions used in the `build.cake` for the driver